### ISSUE TITLE
/quests quit [SUGGESTION]

### ISSUE THREAD

**Issue by agoniaprivat** [PikaMug/Quests#2279]
### What does your feature do?

If config.yml is set to max-quests: 1, removes the requirement to type in the quest name when doing /quests quit.

### How would people use it?

Would stop a lot of confusion for players, since we only allows one quest a time.

**Comment by PikaMug** [comment]
Thanks for the suggestion. I'd consider a pull request for this, but personally I'd rather the player be absolutely certain of which quest it is they're quitting.

**Comment by agoniaprivat** [comment]
> Thanks for the suggestion. I'd consider a pull request for this, but personally I'd rather the player be absolutely certain of which quest it is they're quitting.

Yeah but make so they still have to type "yes" or "no" in chat 👍 

Just that they have to put the quest name in the quit command, gets a little bit weird.

**Comment by HorusApl** [comment]
MyCommand (or similar plugin) can do this. Placeholders can execute themselves via a console command, so doing something like: /questadmin quit $player %quests_player_current_quest_names% would work for your purposes.

**Comment by PikaMug** [comment]
@agoniaprivat Note that the yes/no prompt can be disabled (config.yml `confim-abandon`).

@HorusApl I think they want the player to run the command, but otherwise that's a good solution.

**Comment by HorusApl** [comment]
When using MyCommand, it can execute commands by the player as if they had console level perms. So anything you set there will execute even if it's a regular person. Quick example if they decide to use. $player is the internal placeholder that MyCommand uses for the player that can the command. So in this example, it will do exactly as they suggested, but with /quitquest being a shorthand of that

questsalt5: # just the ID that it needs
  command: /quitquest  # Command the player runs
  type: RUN_CONSOLE
  runcmd:
  - /questadmin quit $player %quests_player_current_quest_names%

**Comment by PikaMug** [comment]
HorusApl seems to have provided a good workaround. There's also the fact that `/questadmin give <player> <quest>` can override the max-quests setting (which is intentional), further complicating a solution. So unless someone submits an all-encompassing pull request, this issue is closed.
